bucket/oracle-instant-client.json

```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://www.oracle.com/database/technologies/instant-client.html",
+    "description": "Oracle Instant Client enables applications to connect to a local or remote Oracle Database for development and production deployment.",
+    "license": "https://www.oracle.com/downloads/licenses/instant-client-lic.html",
+    "version": "19.3.0.0.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.oracle.com/otn_software/nt/instantclient/19300/instantclient-basic-windows.x64-19.3.0.0.0dbru.zip",
+            "hash": "ffceffb0e0024623cbcf5e48409c9c43c36bd117052451cce09faf73325d9931"
+        },
+        "32bit": {
+            "url": "https://download.oracle.com/otn_software/nt/instantclient/19300/instantclient-basic-nt-19.3.0.0.0dbru.zip",
+            "hash": "05d608ce4ee67a3285c4fd4234fb3cfbd5391c48634b10128bea9d8c36cf8456"
+        }
+    },
+    "suggest": {
+       "Microsoft Visual Studio 2017 Redistributable": "vcredist2017"
+    },
+    "extract_dir": "instantclient_19_3",
+    "env_add_path": ".",
```

Comment:
This is really needed? Why do not use bin?

Reply:
There are a series of DLLs that need to be in the `PATH` in order for Oracle applications to work. According to the documentation (https://docs.oracle.com/en/database/oracle/oracle-database/19/lnoci/instant-client.html#GUID-7D65474A-8790-4E81-B535-409010791C2F)

> On Windows, set PATH to the instantclient_18_1 directory.

You could specify exactly the files that need to be on the `PATH`, but I don't know enough of the specifics on this and could not find documentation detailing it either.